### ISSUE TITLE
Cant install integration

### ISSUE THREAD

**Issue by kurtSejr** [astrandb/miele#468]
### The problem

My integration has been running for some months without problems, but since 2024.4 i HA it's stop working "Cant configure" is the error message. I deleted my integration, requested new keys from Miele - but with same result. When I look in the documentation, it seems that other users have had same issue, but has managed to get it working again with a new installation.... - but not me.

### Version of miele integration having the issue?

2024.3.0 according o HACS

### Version of Home Assistant Core having the issue?

2024.4.3

### Anything in the logs that might be useful for us?

```txt
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 551, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/Miele/__init__.py", line 229, in async_setup_entry
    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1860, in async_forward_entry_setups
    await integration.async_get_platforms(platforms)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1145, in async_get_platforms
    import_future.result()
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1145, in async_get_platforms
    import_future.result()
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1145, in async_get_platforms
    import_future.result()
  [Previous line repeated 6 more times]
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1133, in async_get_platforms
    platforms.update(self._load_platforms(platform_names))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1060, in _load_platforms
    platform_name: self._load_platform(platform_name)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1232, in _load_platform
    cache[full_name] = self._import_platform(platform_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1264, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/Miele/vacuum.py", line 9, in <module>
    from homeassistant.components.vacuum import (
ImportError: cannot import name 'VacuumEntityDescription' from 'homeassistant.components.vacuum' (/usr/src/homeassistant/homeassistant/components/vacuum/__init__.py)
```

**Comment by astrandb** [comment]
The last line in your log indicates that you are running an old version of the integration. This deprecation was fixed in v2024.1.0 of the integration and released before HA 2024.2.0 was released where the error otherwise would have appeared.

So please make sure that you have the latest version of the integration installed.

**Comment by kurtSejr** [comment]
SOLVED
Found! - I had two folders, a "Miele" and a "miele" folder - deleted "Miele" folder, and now it seems to be working
